charts/catalog-api/templates/secret-jobs.yaml

```
@@ -20,3 +20,24 @@ data:
   {{- with .Values.dgctlStorage.secretKey }}
   dgctlStorageSecretKey:  {{ . | b64enc }}
```

И тут, если не трудно, давай поправим.

```suggestion
  dgctlStorageSecretKey: {{ . | b64enc }}
```